src/test/regress/expected/pg14.out

```diff
@@ -226,5 +226,48 @@ SELECT attname || ' ' || attcompression FROM pg_attribute WHERE attrelid::regcla
  {"a p","a p","b ","b "}
 (2 rows)
 
+-- ALTER STATISTICS .. OWNER TO CURRENT_ROLE errors out
+CREATE TABLE st1 (a int, b int);
+CREATE STATISTICS role_s1 ON a, b FROM st1;
+-- this should work fine since it's not distributed
+ALTER STATISTICS role_s1 OWNER TO CURRENT_ROLE;
+-- this should error out
```

Comment:
it says this should error out, but where is the error? is this outdated comment maybe?